tcav/tcav_examples/image_models/imagenet/download_and_make_datasets.py

```diff
@@ -0,0 +1,93 @@
+import subprocess
+import os
+import argparse
+from tensorflow.io import gfile
+import imagenet_and_broden_fetcher as fetcher
+"""
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
```

Comment:
move copyright and file-level documentation to top of file like in your other files

Reply:
Done